File: Examples/SnapinsSDKExample/app/src/javaImplDebug/java/com/salesforce/snapinssdkexample/activities/VersionActivity.java
package com.salesforce.snapinssdkexample.activities;

import android.os.Bundle;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.salesforce.android.chat.core.BuildConfig;
import com.salesforce.snapinssdkexample.R;

/**
 * Activity to display the versions of the snap-ins SDKs
 */
public class VersionActivity extends AppCompatActivity {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_version);

        populateVersionText();
    }

    private void populateVersionText() {
        ((TextView) findViewById(R.id.chat_version)).setText(BuildConfig.VERSION_NAME);
        ((TextView) findViewById(R.id.common_version)).setText(com.salesforce.android.service.common.ui.BuildConfig.VERSION_NAME);
    }
}

File: Examples/SnapinsSDKExample/app/src/kotlinImplDebug/java/com/salesforce/snapinssdkexample/activities/VersionActivity.kt
/*
 * Copyright (c) 2018, salesforce.com, inc.
 * All rights reserved.
 * Licensed under the BSD 3-Clause license.
 * For full license text, see LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */

package com.salesforce.snapinssdkexample.activities

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.salesforce.snapinssdkexample.R

/**
 * Activity to display the versions of the snap-ins SDKs
 */
class VersionActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_version)

        populateVersionText()
    }

    fun populateVersionText() {
        findViewById<TextView>(R.id.chat_version).text =
            com.salesforce.android.chat.core.BuildConfig.VERSION_NAME
        findViewById<TextView>(R.id.kb_version).text =
            com.salesforce.android.knowledge.core.BuildConfig.VERSION_NAME
        findViewById<TextView>(R.id.cases_version).text =
            com.salesforce.android.cases.BuildConfig.VERSION_NAME
        findViewById<TextView>(R.id.common_version).text =
            com.salesforce.android.service.common.ui.BuildConfig.VERSION_NAME
    }
}
